tests/pyfunc/test_spark.py

```diff
@@ -1300,6 +1302,21 @@ def predict(self, context, model_input, params=None):
         assert res["res"][0] == ("string")
 
 
+def test_spark_df_schema_inference_for_map_type(spark):
+    data = [{"a": {"a": 1, "b": 2}, "b": ["a", "b"], "c": "c", "d": {"e": ["e", "e"]}}]
```

Comment:
```suggestion
    data = [
        {
            "a": {"a": 1, "b": 2},
            "b": ["a", "b"],
            "c": "c",
            "d": {"e": ["e", "e"]},
        }
    ]
```

Reply:
which column is `Map`? "a" and "d"?

Reply:
`black` formats this like this, otherwise the pre-commit hook can't pass 😓

Reply:
Yes, "a" and "d"

Reply:
```suggestion
    data = [
        {
            "map1": {"a": 1, "b": 2},
            "arr": ["a", "b"],
            "str": "c",
            "map2": {"e": ["e", "e"]},
        }
    ]
```

might be more helpful.